db/fixtures/miq_product_features.yml

```diff
@@ -364,6 +364,10 @@
       :description: Refresh Cloud Providers
       :feature_type: control
       :identifier: ems_cloud_refresh
+    - :name: Re-check Authentication Status 
+      :description: Re-check Authentication Status of Cloud Providers
+      :feature_type: control
+      :identifier: ems_cloud_recheck_auth_status 
```

Comment:
@jrafanie This feature is for all ems types. I was just taking a "test drive" with `ems_cloud`...so once we are all good with the changes for `ems_cloud`, I would be adding this feature for the other ems types.

I agree that the feature is generic, however in the .yml we would have to repeat this feature individually in each ems section... so that would be `ems_infra_recheck_auth_status`, `ems_container_recheck_auth_status` etc